test/steps/crypto/3_then.js

```diff
@@ -17,6 +17,26 @@ import lisk from 'lisk-js';
 import cryptoInstance from '../../../src/utils/cryptoModule';
 import { getFirstQuotedString } from '../utils';
 
+export function theVerifiedMessageShouldBeReturned() {
```

Comment:
This is a misleading function name: the message isn't returned, just the verification.